CONTRIBUTING.md

```diff
@@ -66,15 +67,23 @@ TIP: if you want you can use prefix eg "CSS" or "JS" for branch naming and in yo
 - make sure to use only class and id
 - Try to use ids or combination of selectors so we won't mix our styles
 - If you decide to use classes use prefix of your task which you are working on eg  `footer-link` `nav-search-button`
-- Avoid tag name like main or section or h1 or p this can mix your styles with other styling or you can also use inline styling in your elements
+- Avoid tag name like `main` or `section` or `h1` or `p` this can mix your styles with other styling

```

Comment:
Avoid tag `p`? We need paragraphs. 
Also please add to avoid tag `header` and `footer`.